src/test/java/com/puppycrawl/tools/checkstyle/XpathFileGeneratorAuditListenerTest.java

```diff
@@ -49,23 +49,35 @@
     /** OS specific line separator. */
     private static final String EOL = System.getProperty("line.separator");
 
+    private LocalizedMessage message1;
+
+    private LocalizedMessage message2;
+
+    private LocalizedMessage message3;
+
+    private LocalizedMessage message4;
+
     @Before
```

Comment:
Should this be `BeforeClass`? Why do we need to run this before every test?

Reply:
done